sql/catalyst/pom.xml

```diff
@@ -114,6 +114,11 @@
       <version>2.7.3</version>
       <type>jar</type>
     </dependency>
+    <dependency>
+      <groupId>${hive.group}</groupId>
+      <artifactId>hive-exec</artifactId>
```

Comment:
Why catalyst module needs Hive dependency? We shouldn't add them

Reply:
to import  org.apache.hadoop.hive.ql.session.SessionState

Can spark-sql have dependency on hive ? .  If not then I need to do with reflection or will find other solution

Reply:
No, we're trying to get rid of hive dependencies. Shouldn't bring it back.

Reply:
The problem is specific to CLI. let's scope the fix to that if possible.

Reply:
I ve updated the code, can you review it